charts/langsmith/templates/_helpers.tpl

```diff
@@ -200,6 +200,8 @@ Template containing common environment variables that are used by several servic
     secretKeyRef:
       name: {{ include "langsmith.clickhouseSecretsName" . }}
       key: clickhouse_tls
+- name: CLICKHOUSE_CLUSTER
+  value: {{ .Values.config.clickhouseCluster }}
```

Comment:
can we do .Values.clickhouse.external.clickhouseCluster(where all the other params live?)

Also need to add to the .Values file